src/DeFuncto.Tests/Core/Types/Du6/Match.cs

```diff
@@ -1,75 +1,77 @@
 ﻿using DeFuncto.Extensions;
+using FsCheck;
+using FsCheck.Xunit;
 using Xunit;
 using static DeFuncto.Prelude;
 
 namespace DeFuncto.Tests.Core.Types.Du6
 {
     public class Match
```

Comment:
For all of these cases, add a third input for the ones that are not being matched.